src/styles/components/overrides/_settings.scss

```
@@ -517,6 +597,28 @@
+  color: var(--text-secondary);
+  font-size: $font-size-small;
+  line-height: 1.5;
+  margin: 0;
+
+  code {
+    background: var(--bg);
+    border-radius: $radius-sm;
+    padding: 0.15em 0.4em;
+    font-size: 0.9em;
+    color: var(--text-primary);
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)